.travis.yml

```diff
@@ -0,0 +1,34 @@
+addons:
+  apt:
+    packages:
+    - libgmp-dev
+before_install:
+- sh script/install-stack.sh
+cache:
+  directories:
+  - "${HOME}/.local/bin"
+  - "${HOME}/.stack"
+  - ".stack-work"
+env:
+  global:
+  - CASHER_TIME_OUT=1000
+  - BUILD="elmi-to-json-${TRAVIS_TAG}-${TRAVIS_OS_NAME}.tar.gz"
```

Comment:
Just noting that all travis builds will produce 64-bit binaries.

Reply:
I think travis doesn't have 32bit support right?

Reply:
correct, it does not;  (I've read it's possible to build 32-bit linux on travis by using docker or by using a complicated chroot)